recipe/meta.yaml

```diff
@@ -30,20 +34,70 @@ requirements:
     - anyio
     - idna
     - sniffio
+    - brotli
```

Comment:
```suggestion
```

it's optional? or?

Reply:
It feels like it's one of those where is should be optional but maybe isn't.

The code will try to load `brotli` or `brotlicffi` or complain: see the load in [httpx/_compat.py](https://github.com/encode/httpx/blob/0.27.0/httpx/_compat.py#L11-L17) and barf in [httpx/_decoders.py](https://github.com/encode/httpx/blob/0.27.0/httpx/_decoders.py#L99-L105) if brotli is used anywhere.

So, maybe optional?

Reply:
We've gone for optional, for the record.
We actually need `brotli-python` for any of it to work anyway.